integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -64,6 +64,13 @@ spec:
 {{ toYaml .Values.worker.nodeSelector | trim | indent 8  }}
       {{- else if .Values.nodeSelector }}
 {{ toYaml .Values.nodeSelector | trim | indent 8  }}
+      {{- end }}
+      tolerations:
+      {{- if .Values.worker.tolerations }}
+{{ toYaml .Values.worker.tolerations | trim | indent 8  }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+{{ toYaml .Values.tolerations | trim | indent 8  }}
```

Comment:
@jiacheliu3 Is this what you meant by union? This will include duplicate definitions if the same toleration is in both `.Values.tolerations` and `.Values.xxx.tolerations` but I think that's on the user if they do that. Also I tested duplicate toleration definitions and I don't think it matters to Kubernetes anyway

Reply:
Exactly this is what I meant